pkg/os/shell/shell_windows_test.go

```diff
@@ -0,0 +1,41 @@
+package shell
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDetect(t *testing.T) {
+	defer func(shell string) { os.Setenv("SHELL", shell) }(os.Getenv("SHELL"))
+	os.Setenv("SHELL", "")
+
+	shell, err := Detect()
+
+	assert.Equal(t, "powershell", shell)
```

Comment:
@guillaumerose I am thinking to use `Contains` here, this way we don't have to worry if it is powershell or cmd as long as it is part of supported shell?
```
assert.Contains(t, supportedShell, shell)
```

Reply:
Good suggestion! I did this and remove the 2nd test. It was bringing more information.